lib/src/async_import_cache.dart

```diff
@@ -122,6 +126,12 @@ class AsyncImportCache {
       {AsyncImporter? baseImporter,
       Uri? baseUrl,
       bool forImport = false}) async {
+    if (isBrowser &&
+        (baseImporter == null || baseImporter is NoOpImporter) &&
+        _importers.isEmpty) {
+      throw "Custom importers are required to `@use` or `@import` when compiling in the browser.";
```

Comment:
Nit: I'd say something like "to load stylesheets" here, in case users get confused and thing that other structures like `@forward` or `meta.load-css()` are allowed.

Reply:
Fixed in c3b373eeac4953f592dc8ff0c13771131ccbe4ec